zuul.d/tempest_periodic.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- job:
```

Comment:
[Suggestion] Add a description this this job explaining what it's doing and answering the other reviewers questions.

Reply:
from the added inline comments in code I now understand that 'master' refers to rdo release for current-podified. So +1 to having added those